cmd/scollector/collectors/sql_windows.go

```diff
@@ -30,20 +30,33 @@ func init() {
 	c_replica_server.init = wmiInit(c_replica_server, func() interface{} { return &[]Win32_PerfRawData_MSSQLSERVER_SQLServerAvailabilityReplica{} }, `WHERE Name <> '_Total'`, &sqlAGQuery)
 	collectors = append(collectors, c_replica_server)
 
-	var hostname, _ = os.Hostname()
+	hostname, _ = os.Hostname()
```

Comment:
We can just switch to `util.Hostname` here. It gets set and correctly cleaned and is used in other parts of scollector.